Sources/Fuzzilli/Lifting/FuzzILLifter.swift

```diff
@@ -233,17 +233,25 @@ public class FuzzILLifter: Lifter {
             w.decreaseIndentionLevel()
             w.emit("EndIf")
 
-        case is BeginSwitch:
+        case let op as BeginSwitch:
             w.emit("BeginSwitch \(input(0))")
-            w.emit("DefaultCase")
+            if op.firstCaseIsDefaultCase {
+                w.emit("DefaultCase")
+            } else {
+                w.emit("BeginSwitchCase \(input(1))")
+            }
             w.increaseIndentionLevel()
 
         case let op as BeginSwitchCase:
             if !op.fallsThrough {
                 w.emit ("Break")
             }
             w.decreaseIndentionLevel()
-            w.emit("BeginSwitchCase \(input(0))")
+            if op.isDefaultCase {
```

Comment:
Same here, I think this should just be one line with zero or one inputs and maybe a note if the previous case falls through or not. E.g. `BeginSwitchCase v4 previousCaseFallsThrough` for a "normal" case with fallthrough or `BeginSwitchCase` for a default case with no fallthrough